tests/operators/docker_operator.py

```diff
@@ -158,6 +159,82 @@ def test_on_kill(self):
 
         client_mock.stop.assert_called_with('some_id')
 
+    @unittest.skipIf(mock is None, 'mock package not present')
```

Comment:
don't skip

Reply:
Ok. Should we bother removing it from the existing tests in `docker_operator.py`?

Reply:
Done for all.